### PR TITLE
Update UnitTest projects to use .netcore 3.1

### PR DESCRIPTION
**Pull request by FermJacob**
Specflow unit tests are in a different task

### CODE REVIEW

File: Framework/Base.sln [hunk 1]
```diff
@@ -42,7 +42,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EmailUnitTests", "EmailUnit
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SpecFlowExtension", "SpecFlowExtension\SpecFlowExtension.csproj", "{26D35A87-D2F4-4A04-931A-EF2DBEA9EBB5}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CoreUnitTests", "CoreUnitTests\CoreUnitTests.csproj", "{D3E59B9F-445E-4484-9B2F-B3FE3ED82CE5}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FrameworkUnitTests", "CoreUnitTests\FrameworkUnitTests.csproj", "{D3E59B9F-445E-4484-9B2F-B3FE3ED82CE5}"
```
Comment (TroyWalshProf) [inline Framework/Base.sln:45]: Good you changed the project name but the folder name should change as well - Very weird to have the famework tests in CoreUnitTests folder

**Comment by TroyWalshProf** [comment]
Suggest adding selenium screenshot tests to Framework tests.  We are loosing a little coverage because all the core tests need to be png

**Comment by FermJacob** [comment]
@TroyWalshProf Only other thing I can think of is making sure app.config and appsettings.json are correct with their respective projects.  I think they are off, but still work because of how config gets loaded.  Should we change it before merging this?

File: Framework/WebServiceUnitTests/WebServiceUnitTests.csproj [hunk 2]
```diff
@@ -17,6 +17,7 @@
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.16.1" />
     <PackageReference Include="System.Buffers" Version="4.5.1" />
+    <PackageReference Include="System.Drawing.Common" Version="4.7.0" />
```
Comment (TroyWalshProf) [inline Framework/WebServiceUnitTests/WebServiceUnitTests.csproj:20]: Are you sure this is needed

Comment (FermJacob) [reply Framework/WebServiceUnitTests/WebServiceUnitTests.csproj:20]: Yes, We have a API test that tests an image that this library is required

**Pull request merged**